README.md

```diff
@@ -19,6 +19,24 @@ If you have selected install_dependencies during the prompt, you can skip this p
     conda env create -f environment.yml 
     ```
 
+## Using the package
+
+For installation using pip:
+```sh
+pip install uk-gov-dash-components
+```
+
+or for a specific version:
+```sh
+pip install uk-gov-dash-components~=6.7.0
+```
+
+For installation using conda, paste the following code into the environment configuration file:

```

Comment:
I find the wording of this a bit confusing because although it's for the conda environment, it still uses pip to install it. Maybe we could reword it slightly to say to install it into a conda environment?